onnx/onnxifi_ext.h

```diff
@@ -7,6 +7,11 @@
 extern "C" {
 #endif
 
+static const int ALL_EXT_FUNCTION_NUMBER = 2;
+static const char ALL_EXT_FUNCTION_LIST[] =
```

Comment:
what's the reason to represent all functions in this long char str? and not
char *list[] = {"a", "b"} etc.